src/components/buttons/HdLoaderButton.vue

```diff
@@ -131,23 +173,28 @@ export default {
     loadingState(state) {
       if (state === 'success') {
         this.loadedAmount = 1;
-      }
-      if (state === 'error') {
-        this.transitionsQue.push(this.setErrorState);
+      } else if (state === 'error') {
```

Comment:
Is there a particular reason to add the `else` here?
As `state === 'success'`, and `state === 'error'` are mutually exclusive, the `else` is not needed.
Or we want this to be extra clear?

Reply:
No particular reason, I just checked and It wasn't there before I think it was added by Prettier.